demo/Makefile

```diff
@@ -11,7 +11,7 @@ run: build
 build-snapshot:
 	docker build -t $(IMAGE):RELEASE .
 	mkdir -p snapshot-plugins
-	for p in $$(cat plugins.txt|perl -pe s/:.+//g; cat snapshot-only-plugins.txt); do echo looking for snapshot builds of $$p; for g in org/jenkins-ci/plugins org/jenkins-ci/plugins/workflow org/jenkins-ci/plugins/pipeline-stage-view; do if [ -f ~/.m2/repository/$$g/$$p/maven-metadata-local.xml ]; then cp -v $$(ls -1 ~/.m2/repository/$$g/$$p/*-SNAPSHOT/*.hpi | tail -1) snapshot-plugins/$$p.jpi; fi; done; done
+	for p in $$(cat snapshot-only-plugins.txt); do echo looking for snapshot builds of $$p; for g in org/jenkins-ci/plugins org/jenkins-ci/plugins/workflow org/jenkins-ci/plugins/pipeline-stage-view; do if [ -f ~/.m2/repository/$$g/$$p/maven-metadata-local.xml ]; then cp -v $$(ls -1 ~/.m2/repository/$$g/$$p/*-SNAPSHOT/*.hpi | tail -1) snapshot-plugins/$$p.jpi; fi; done; done
```

Comment:
I expected this bug TBH :)

So, as any developer, I have plenty of plugin snapshots in my local Maven repository which I just want to ignore. Is it supposed that I have to clean my local `.m2/repository` if I want to use only released versions?

> It is the only way to test the behavior of unreleased changes in the image

If you want to test plugin `my-plugin2.3-SNAPSHOT`: run `mvn clean install`, add `my-plugin` to `snapshot-only-plugins.txt` and rebuild the image. I think it worked for me. Am I missing something?